nextjs-13/src/stylesheets/screens/_view-newsletter.scss

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
this extra line break can. be removed

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/127/commits/66961365348990a90cc5eb7f779ddc90f7be62d5)